composer.json

```diff
@@ -76,8 +76,8 @@
         "spryker-feature/shared-carts": "^202212.0",
         "spryker-feature/shipment": "^202212.0",
         "spryker-feature/shopping-lists": "^202212.0",
-        "spryker-feature/spryker-core": "^202212.0",
-        "spryker-feature/spryker-core-back-office": "^202212.0",
+        "spryker-feature/spryker-core": "dev-master as 202304.0",
+        "spryker-feature/spryker-core-back-office": "dev-master as 202304.0",
```

Comment:
It seems like `spryker-shop/security-blocker-page` should also be updated according to https://github.com/spryker/spryker-shop/pull/2332